Dvorak-Right-Handed/KeyboardView.swift

```diff
@@ -176,7 +193,7 @@ class KeyboardView: UIView {
         let _ = allKeys.map({$0.titleLabel?.font = myFont})
         self.globeKey.setTitle("🌐", for: .normal)
         self.backspaceKey.setTitle("⌫", for: .normal)
-        self.enterKey.setTitle("⏎", for: .normal)
+        self.enterKey.setTitle("enter", for: .normal)
```

Comment:
"Enter" or "ENTER"?

Reply:
Its lower case to match "return" on standard apple keyboard

Reply:
I'm using "enter" instead of "return" because some apps, ie. snapchat, replace "return" with "send" and I thought "enter" was a happy medium. it can represent both 'return' and 'send'.